technology-considerations/droid_vs_siegfried.md

```diff
@@ -0,0 +1,22 @@
+## Droid vs Siegfried
+
+We need software to carry out the file format checks on the files uploaded to TDR. There are two choices, [Siegfried](https://github.com/richardlehane/siegfried) and [Droid](https://github.com/digital-preservation/droid)
+
+### Siegfried
+#### Advantages
+* Faster than Droid
+* Outputs results to json
+
+##### Disadvantages
+* Doesn't report multiple patterns correctly. There is as [issue](https://github.com/richardlehane/siegfried/issues/146) where the maintainer says they will fix it but we don't know when that will be.
+
+### Droid
+#### Advantages 
+* Reports files that contain multiple identification patterns correctly.
+* Is developed by TNA so we have a lot more control over its development.
+
+#### Disadvantages
+* Slower than Siegfried.
```

Comment:
Do you have any numbers to quantify this? e.g. do you know how long the Siegfried and DROID Lambdas take for the same file?

It would be good to know in case we ever have to revisit this decision for performance reasons.

Reply:
I didn't measure the time when the lambda was running but I've run both locally on the same file and I've put some numbers in there.